### DISCUSSION TITLE
What will happen if part of AggregateCommits goes wrong?

### DISCUSSION THREAD

**Discussion by joshua-ne** [filecoin-project/lotus#6607] [General Q&A] [answered]
Could someone give us some hints on what will happen if the PoRep proof of some sectors got wrong in an AggregateCommits message? Say we have 10 PoRep aggregated in one message (sector 1-10), and for sector 1, the preCommit has expired and for sectors 2 and 3, the proofs are somehow wrong, what will happen to the execution of this message?
1. Will the message return ok?
2. How will the lotus-miner know the execution status of each sector?
3. Will sectors 4-10 turn to a state of Proving?
4. In StateMachine, what will happen to sectors 1-3? Do they return to a retrying status?

Any answers or guidance to specific code sessions would be greatly appreciated. Many thanks! @cryptonemo @nicola @ninitrava @ZenGround0

**Comment by nicola** [comment, marked as answer]
Currently if one of the proofs is invalid the whole message fails.
